jaxrs-tck/pom.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <!--
 
-    Copyright (c) 2020 Markus Karg. All rights reserved.
+    Copyright (c) 2020, 2021 Markus Karg. All rights reserved.
```

Comment:
The EF allows us to use the short syntax with a single date only. That would spare us from the clutter of frequently changed header lines. I would really love it if we could all agree that as it is not enforced by the EF we do stop using the two-dates variant.

Reply:
I'll defer this to somebody with more IP/Legal experience. Personally, I'd prefer that we didn't need to put or update a copyright date at all, but I think the Eclipse Foundation requires it.

Reply:
The EF's rules are clear and publicly available: They demand ONLY the initial year, but do NOT require the second, updated year. This IS the official EF statement. So if nobody wants to have it updated, the EF is fine if we stick with the original single-year variant.